### ISSUE TITLE
Default block types not working

### ISSUE THREAD

**Issue by timmyg** [madebymany/sir-trevor-js#442]
It seems as though Heading and Quote aren't available as block types in the UI (see examples/index.html)

**Comment by timmyg** [comment]
im looking into this...

**Comment by raffij** [comment]
Yeah. I'll revert a change and a better way to deal with this. Working on
some sir Trevor updated tomorrow.

**Comment by timmyg** [comment]
Yea. I'm not sure why it's not working...

**Comment by raffij** [comment]
Not really sure on what config we will use in the future so use the following for now before initialising your editor.

SirTrevor.Blocks.Heading.prototype.toolbarEnabled = true;
SirTrevor.Blocks.Quote.prototype.toolbarEnabled = true;
SirTrevor.Blocks.Text.prototype.toolbarEnabled = true;

**Comment by timmyg** [comment]
Nice!!

**Comment by nilsi** [comment]
Still had to do this for text and heading block to show up. Not sure if this is intended. Just wanted to point it out.
